src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/handlers/VarEventHandler.java

```diff
@@ -25,24 +25,18 @@
  */
 public abstract class VarEventHandler extends BaseSingleTagEventHandler {
     private final String name;
-    private final boolean isInternal;
 
     @TruffleBoundary
-    public static boolean isIdentifierInternal(String id) {
-        return id.equals(":switch") || (id.startsWith("<") && !id.equals("<this>"));
+    public static boolean isThis(String id) {
+        return id.equals("<this>");
```

Comment:
This method seems to be unused. Remove?

Reply:
yes, it is now taken care by graal.js to exclude internal accesses so this method is no longer useful.

Reply:
do we need to return "this" instead of ```<this>``` to the user as identifier name, if so we need isThis to check.

Reply:
Since we are already returning `this` (the name is demangled in Graal.js) I removed the unused check.